lib/_tls_wrap.js

```diff
@@ -608,6 +623,21 @@ TLSSocket.prototype._init = function(socket, wrap) {
       // Remover this listener since its no longer needed.
       this.removeListener('newListener', newListener);
     }
+
+    ssl.onkeylog = onkeylogclient;
+
+    // Only call .onkeylog if there is a session listener.
+    this.on('newListener', keylogNewListener);
+
+    function keylogNewListener(event) {
+      if (event !== 'keylog')
+        return;
+
+      ssl.enableKeylogCallback();
+
+      // Remover this listener since its no longer needed.
```

Comment:
`s/Remover/Remove/`, `s/its/it's/`

Reply:
Thanks, I have corrected the original line too.